packages/ui/src/views/connectors/utils.tsx

```diff
@@ -0,0 +1,28 @@
+import { Icon } from '@components/icon'
+import { Logo } from '@components/logo/logo'
+import { SimpleIcon } from 'simple-icons'
+
+import { ConnectorConfigType } from './types'
+
+export const mapConnectorTypeToLogoName = (type: ConnectorConfigType): string | undefined => {
+  switch (type) {
+    case 'Github':
+      return 'github'
+    case 'Gitlab':
+      return 'github'
+    case 'Bitbucket':
+      return 'bitbucket'
+    case 'Jira':
+      return 'jira'
+    case 'K8sCluster':
+      return 'kubernetes'
+    default:
+      return undefined
+  }
+}
+
+export const getConnectorLogo = (connectorType: ConnectorConfigType): SimpleIcon | React.ReactNode => {
```

Comment:
Moved this out to a separate util for reusability, this could be used on Create/Edit connector drawer as well.